src/vue/common/NoDataMessage.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="no-data-message">
     <i v-if="iconName" :class="`mdi mdi-${iconName}`" />
-    <h2>{{ msgTitle }}</h2>
-    <p>{{ msgMessage }}</p>
+    <h2>{{ titleId | globalize(titleIdArgs) }}</h2>
+    <p>{{ messageId | globalize(messageIdArgs) }}</p>
```

Comment:
looks shitty we definitely need to transmit this responsibility to the elements that use component. better to do this after all the main work done so let’s create a task for it

Reply:
https://app.asana.com/0/894507180698157/1106753753322063